### ISSUE TITLE
would prefer not to write output file

### ISSUE THREAD

**Issue by nickperkinslondon** [DataGarage/node-xlsx-json#17]
The data is passed to the callback, which is what I use -- I have no need to write a json file to disk -- but it seems to be required.  ( awesome package thank you very much! )

**Comment by steven0lisa** [comment]
You can set output null

**Comment by nickperkinslondon** [comment]
oh, you can?....i guess i just tried omitting it ( leaving it undefined )...and i got some horrible error or something --  does <undefined> do the same thing as null?

**Comment by qualitycode** [comment]
No,  undefined does NOT do the same thing as null, as I found out the hard way.  It took a long time to realize that I have to set output as null.  We should not have to send in output at all if it is not being used.
At the least, it would be helpful to document this in the readme file.

**Comment by mikkelking** [comment]
I also set the output file to `/dev/null`, which is succcessful
